deployment/serviceCmd.py

```diff
@@ -61,22 +63,25 @@ def add_arguments(parser):
     def process_args(self, args):
         if args.kube_config_path is not None:
             args.kube_config_path = os.path.expanduser(args.kube_config_path)
-        return args.service_list
+
```

Comment:
Try to use https://docs.python.org/2.7/library/argparse.html#mutual-exclusion?

Reply:
> Try to use https://docs.python.org/2.7/library/argparse.html#mutual-exclusion?

Fixed, thx